code/modules/reagents/reagent_containers/hypospray.dm

```diff
@@ -175,6 +175,11 @@
 	desc = "A modified stimulants autoinjector for use in combat situations. Has a mild healing effect."
 	list_reagents = list(/datum/reagent/medicine/stimulants = 10, /datum/reagent/medicine/omnizine = 10)
 
+/obj/item/reagent_containers/hypospray/medipen/stimpack/elemetx
+	name = "element x medipen"
+	desc = "Contains a rare sample of Element X, a mysterious substance that, when injected into the bloodstream, alters one's DNA and activates all positive mutations."
```

Comment:
I can see this being easilly confused with giving someone all positive mutations rather than just the ones thay already have